module/attribute/src/Domain/ValueObject/AttributeCode.php

```diff
@@ -13,7 +13,11 @@
 
 class AttributeCode extends AbstractCode
 {
-    public const PATTERN = '/^([a-zA-Z0-9_]+)$/';
+    public const NOT_SYSTEM_ATTRIBUTE_PATTERN = '/^(?!'.self::SYSTEM_ATTRIBUTE_PREFIX.')[a-zA-Z0-9_]+$/';
```

Comment:
it's just a pattern as it was. SystemAttributeCode should have it's own definition and validate by it